src/servicesHost.ts

```diff
@@ -789,12 +794,28 @@ export function makeSolutionBuilderHost(
       );
       instance.typeReferenceResolutionCache?.update(instance.compilerOptions);
       instance.moduleResolutionCache?.update(instance.compilerOptions);
+
+      if (options) {
+        // The `configFilePath` is the same value that is used as the `project` parameter of
+        // `getCustomtransformers` below.
+        const project = options.configFilePath?.toString();
```

Comment:
is the `toString` necessary?

Reply:
isn't it always a string?

Reply:
You're right.  I don't know what I was thinking.  It's a string.